Dockerfile

```diff
@@ -1,26 +1,28 @@
-FROM ghcr.io/tiiuae/fog-ros-baseimage-builder:v2.1.0 AS builder
+# Given dynamically from CI job.
+FROM --platform=${BUILDPLATFORM:-linux/amd64} ghcr.io/tiiuae/fog-ros-sdk:v3.0.1-${TARGETARCH:-amd64} AS builder
 
-COPY . /main_ws/src/
+# Must be defined another time after "FROM" keyword.
+ARG TARGETARCH
 
-# this:
-# 1) builds the application
-# 2) packages the application as .deb in /main_ws/
+# SRC_DIR environment variable is defined in the fog-ros-sdk image.
+# The same workspace path is used by all ROS2 components.
+# See: https://github.com/tiiuae/fog-ros-baseimage/blob/main/Dockerfile.sdk_builder
+COPY . $SRC_DIR/mocap_pose
 
-RUN /packaging/build.sh
+# Tar directories so they are easier to handle when doing installation.
+RUN /packaging/build_colcon_sdk.sh ${TARGETARCH:-amd64}
+# Even though it is possible to tar the install directory for retrieving it later in runtime image,
+# the tar extraction in arm64 emulated on arm64 is still slow. So, we copy the install directory instead
 
-#  ▲               runtime ──┐
-#  └── build                 ▼
-
-FROM ghcr.io/tiiuae/fog-ros-baseimage:v2.1.0
-
-HEALTHCHECK --interval=5s \
```

Comment:
Health checks were removed with this commit.

I noticed from Marita's shared logs:
> 🟨 $hypervisor.mocap-pose | no health check specified

http://100.117.239.56/logs/2023-10-23_10_46-back-12m0s-ghaf/health.txt

I guess this is an accidental regression and not on purpose.